website/client-old/js/components/groupTasks/groupTasksController.js

```diff
@@ -158,10 +158,12 @@ habitrpg.controller('GroupTasksCtrl', ['$scope', 'Shared', 'Tasks', 'User', '$ro
        Tasks.updateTask(task._id, task);
      };
 
-    $scope.checkGroupAccess = function (group) {
+    $scope.checkGroupAccess = function (group, sup) {
       if (!group || !group.leader) return true;
-      if (User.user._id !== group.leader._id) return false;
-      return true;
+      var userId = User.user._id;
+      var leader = group.leader._id === userId;
+      var userIsManager = !!group.managers[userId];
```

Comment:
Can this be changed to be more like `isManager = Boolean(group.managers[user._id]);` for consistency with that code?